static/src/javascripts/lib/geolocation.js

```diff
@@ -14,6 +14,11 @@ const daysBeforeGeolocationRefresh = 10;
 
 const getFromStorage = (): string => storage.get(storageKey);
 
+const geoLocationEstablishedEvent = 'geolocation:established';
```

Comment:
I would go for consistency with the capitalisation - since there are existing examples like `geolocation` I would say adding examples like `geoLocation` will trip people up later